projects/core/src/checkout/store/reducers/checkout.reducer.ts

```diff
@@ -3,6 +3,10 @@ import { DeliveryMode, Order } from '../../../model/order.model';
 import { CheckoutStepsState } from '../checkout-state';
 import { CheckoutActions } from './../actions/index';
 
+const ONE = 1;
+const TWO = 2;
+const THREE = 3;
```

Comment:
Ok, here we can really have an exception for put numeral in variable
```suggestion
const [FIRST_STEP, SECOND_STEP, THIRD_STEP] = [1,2,3];

```